rollup.config.js

```diff
@@ -82,7 +82,9 @@ export default {
     // only use google analytics on production builds
     replace({
       __GOOGLE_ANALYTICS_ID__:
-        process.env.CONTEXT === "production" && process.env.GOOGLE_ANALYTICS_ID,
+        (process.env.CONTEXT === "production" &&
+          process.env.GOOGLE_ANALYTICS_ID) ||
+        "",
```

Comment:
why do we need to add `|| ""` here?

Reply:
Otherwise `__GOOGLE_ANALYTICS_ID__` gets replaced as `false`, which is not what we want here (since the conditional logic to turn it off altogether for a build looks for an empty string).